python/test/test_cuda_libs.py

```
@@ -0,0 +1,114 @@
+        with tempfile.TemporaryDirectory() as root:
+            package_root = make_pip_cuda_layout(root, {
+                'cublas': ['libcublas.so.12'],
+                'cusolver': ['libcusolver.so.11']})
+            self.assertEqual(_cuda_libs.preload(package_root), [])
+
+    def test_import_resolved_the_cuda_math_libraries(self):
+        # Whatever provided them, the CUDA math libraries libcuvslam.so links against are loaded once cuvslam
+        # imports: this is the regression guard for wheels that neither bundle nor declare them. It runs in a
+        # fresh interpreter, because a cuvslam already imported by another test would load them regardless of
+        # whether importing it still does.
+        probe = subprocess.run([sys.executable, '-c', IMPORT_SIDE_EFFECT_PROBE],
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)